src/Qbead.h

```diff
@@ -150,20 +257,20 @@ class Qbead {
   const uint8_t ix, iy, iz;
   const bool sx, sy, sz;
   float rbuffer[3];
-  float x, y, z, rx, ry, rz; // filtered and raw acc, in units of g
-  float t_acc, p_acc;        // theta and phi according to gravity
+  float rx, ry, rz;         // filtered and raw acc, in units of g
   float T_imu;             // last update from the IMU
 
-  float t_ble, p_ble; // theta and phi
```

Comment:
similarly here, these should be places where the BLE controller can store some variables. It is up to the user whether they will then be put as values for the State object. E.g. an example sketch can simply update the state object to have these values on each iteration of a loop.